pkg/syfs/syfs.go

```diff
@@ -43,20 +43,26 @@ func ConfigDir() string {
 }
 
 func configDir() string {
-	user, err := user.Current()
-	if err != nil {
-		sylog.Warningf("Could not lookup the current user's information: %s", err)
+	configDir := os.Getenv("SINGULARITY_CONFIGDIR")
```

Comment:
Ideally, declare a constant for `SINGULARITY_CONFIGDIR` and use it in the Getenv and any error messages.

We have other code that doesn't do this, but going forward we want to avoid hard coding these in multiple locations.